containers/signing-service/Dockerfile.j2

```diff
@@ -0,0 +1,21 @@
+FROM debian:{{ debian_version }}
+ARG SIGNING_SERVICE_PATH=/usr/lib/signing_service
```

Comment:
`ARG` suggests that the user can set it from the command line when running `docker build` but that's not the case here. We have the path hard-coded in scripts so the container will only work with this one value.

If you want to have variable here, I'd use `ENV` instead. But I think it's better to hard-code it if we don't support changing it.

Reply:
Done.